woocommerce/woocommerce-android
# CardReader connection flow - rename fragments

Parent issue #3725

This PR just renames the following fragments and related classes using AS refactoring tools
- CardReader**Connect**Fragment to CardReader**Detail**Fragment - 55e03e4
- CardReader**Scan**Fragment to CardReader**Connect**Fragment - c351641

I recommend reviewing this PR by commits as it might get confusing otherwise, since we are renaming **Connect** fragment to Detail and Scan fragment to **Connect**

I decided to change the planned structure a bit so we can use just two fragments
-  CardReader**Detail**Fragment - which will be used for both "reader not connect" and "reader connected" states
- CardReader**Connect**Fragment - which will contain the whole connection flow which is consisted of "Scanning","Readers found",  "Connecting", "Error" states/dialogs

I think this will facilitate orientation for other devs and simply the whole flow.  

To test - no reader required
1. Open app settings
2. Tap on "Manage card reader"
3. Tap on "Redirect to CardReaderDetailFragment"
4. Verify you got redirected to CardReaderDetailFragment
4. Tap on "Connect"
5. Verify you got redirected to CardReaderConnectFragment



Update release notes:
